hyperdash/code_runner.py

```diff
@@ -25,9 +25,9 @@ def __init__(self, f, hd_client, parent_logger, *args, **kwargs):
     def wrap(self, f, *args, **kwargs):
         arg_spec = getargspec(f)
         # Make sure function signature can handle injected hyperdash object
-        if "hd_client" in arg_spec.args or arg_spec.keywords:
```

Comment:
I dont know if we want to remain backwards compatible at this point, but we could just support both

Reply:
I think it's a good moment to move forward, since relatively few people know about the experiment API.